macro/src/lib.rs

```diff
@@ -101,23 +97,27 @@ fn gen_bridge(input: ItemMod) -> ItemMod {
     let all_structs = extract_from_mod(&input);
     let (brace, mut new_contents) = input.content.unwrap();
 
-    new_contents.iter_mut().for_each(|c| match c {
-        Item::Struct(s) => {
-            if !s.attrs.iter().find(|a| a.path.to_token_stream().to_string() == "repr").is_some() {
+    new_contents.iter_mut().for_each(|c| {
+        if let Item::Struct(s) = c {
+            if !s
+                .attrs
+                .iter()
+                .any(|a| a.path.to_token_stream().to_string() == "repr")
```

Comment:
note (in case I forget for later, this can't be handled in the current code anyway): for opaque structs we eventually want to be using `repr(transparent)`